packages/radio/src/RadioGroup.ts

```diff
@@ -104,9 +104,7 @@ export class RadioGroup extends FocusVisiblePolyfillMixin(FieldGroup) {
                 const selectedRadio = this.querySelector(
                     `sp-radio[value="${this.selected}"]`
                 ) as Radio;
-                if (!selectedRadio) {
-                    this.selected = '';
-                } else {
+                if (selectedRadio) {
```

Comment:
Just to confirm, is the removal of code in lines 107, 108, and 109 in this PR intended to support the usage scenario demonstrated in this [studio example](https://studio.webcomponents.dev/edit/vUinRWkhayMTAmr9AK9J/src/index.ts?branch=late%40wfVpXdsTYhf9mY2slxW4t2Y2SjC3&p=stories)? However, when I check the demo link, there doesn't seem to be an initially selected radio button still. Am I missing something?

Reply:
It seems studio example is not debuggable. How can I setup a local debuggable project based on the example?

Reply:
This is not to say we should debug there, but to say that you change allows something similar to that. At least I see that as the reason that React doesn't work here. With that in mind, it would be great to add a test for this change based on this code. Maybe we can't wait the whole update cycle to add children, but React is clearly adding children late (or this wouldn't happen) and if this is something we need to support for that use case we should have a test for it.

Reply:
This appears to do it:
```

describe('Radio Group - late children', () => {
    it.only('accepts frame late children', async () => {
        const test = await fixture(html`
            <div>
                <sp-radio value="first">Bulbasaur</sp-radio>
                <sp-radio value="second">Squirtle</sp-radio>
                <sp-radio value="third">Charmander</sp-radio>
                <sp-radio value="fourth">Other</sp-radio>
            </div>
        `);
        const group = document.createElement('sp-radio-group');
        const buttons = [...test.querySelectorAll('sp-radio')] as Radio[];

        test.append(group);
        group.selected = 'first';
        Promise.resolve()
            .then(function () {
                group.append(...buttons);
            });
        await nextFrame();
        await nextFrame();

        expect(group.buttons.length).to.equal(4);
        expect(group.selected).to.equal('first');
    });
});
```
Please help confirm that this FAILS in the old version and PASSES with your update.

Reply:
Thank you so much!